src/tests/integration-test

```diff
@@ -96,7 +96,7 @@ class UDisksTestCase(unittest.TestCase):
     manager = None
 
     @classmethod
-    def init(klass, logfile=None):
+    def init(cls, logfile=None):
```

Comment:
Correct, I believe in many cases these should just be methods on an instance, not class methods.